buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+plugins {
+    `kotlin-dsl`
+}
+
+// kotlinDslPluginOptions.experimentalWarning.set(false)
```

Comment:
Is it needed or should we remove?

Reply:
The comment or the `kotlin-dsl`? The comment isn't needed, and could be removed

Reply:
Without line 5 the following is issued:
```
The `kotlin-dsl` plugin applied to project ':buildSrc' enables experimental Kotlin compiler features.
```

Since we "know" we're using the kotlin dsl, we should probably ignore the warning.